src/components/Headline.js

```diff
@@ -0,0 +1,8 @@
+//@ts-check
+import styled from 'styled-components/macro'
+
+const Headline = styled.h1`
+  font: 300;
+  font-size: 1.75rem;
+`
+export default Headline
```

Comment:
Maybe  a better name could be `Heading` instead of `Headline` as Jerry told us it's the typical naming convention for this.